students/Tianx/Lesson2/charges_calc.py

```diff
@@ -0,0 +1,129 @@
+"""
+Returns total price paid for individual rentals
+
+logging with command line --debug or -d
+
+0: No debug messages or log file.
+1: Only error messages.
+2: Error messages and warnings.
+3: Error messages, warnings and debug messages.
+
+All logs are written to a .log file in this directory as well as the console
+"""
+import argparse
+import json
+import datetime
+import math
+import logging
+
+
+def logger(debug_level):
+    """Setting up logging level"""
+    LOG_FORMAT = '%(asctime)s %(filename)s:%(lineno)-3d %(levelname)s %(message)s'
+    LOG_FILE = 'charges_calc_'+datetime.datetime.now().strftime('%Y-%m-%d')+'.log'
+
+    FORMATTER = logging.Formatter(LOG_FORMAT)
+    FILE_HANDLER = logging.FileHandler(LOG_FILE, mode="w")
+    FILE_HANDLER.setFormatter(FORMATTER)
+
+    CONSOLE_HANDLER = logging.StreamHandler()
+    CONSOLE_HANDLER.setFormatter(FORMATTER)
+    LOGGER = logging.getLogger()
+    LOGGER.addHandler(FILE_HANDLER)
+    LOGGER.addHandler(CONSOLE_HANDLER)
+
+    if debug_level == '0':
+        LOGGER.disabled = True
```

Comment:
Note that with -d 0 your code is still showing logging messages. Try to fix that. Consider the possibility of assigning CRITICAL to level 0:

```
    LOGGER.setLevel(logging.CRITICAL)
    FILE_HANDLER.setLevel(logging.CRITICAL)
    CONSOLE_HANDLER.setLevel(logging.CRITICAL)
```